minitask1/benh/turtlebot_driving.py

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/env python
+import rospy
+import tf
+
+from math import pi as PI, sqrt
+
+from nav_msgs.msg import Path, Odometry
+from geometry_msgs.msg import Twist, PoseStamped, Pose
+
+
+def get_forward_twist():
```

Comment:
There's a simpler way to use the Twist() command here rather than setting each of the variables within it as they stay at 0 anyway.